news/75.bugfix

```diff
@@ -0,0 +1 @@
+Make it work when Git is not installed
```

Comment:
I'd rather consider this a feature rather than a bug fix.

Because typically when you build a project, you are in a dev environment where it is not exotic to request `git` to be installed.